crates/bdk/tests/wallet.rs

```diff
@@ -445,6 +461,16 @@ macro_rules! check_fee {
     }};
 }
 
+/// A floating point assert! that takes an additional third argument `delta`
+/// as a tolerance value when test for equality the first and second argument.
+macro_rules! assert_delta {
+    ($x:expr, $y:expr, $d:expr) => {
+        if !($x - $y < $d || $y - $x < $d) {
+            panic!();
+        }
+    };
+}
+
```

Comment:
This the way I've found to ignore rounding errors when comparing `FeeRate::from_sat_per_vb` after the wallet signed with PKH descriptor.

Reply:
I think once #1141 is in you'll be able to use a different function to compare fee rates like [to_sat_per_kwu](https://docs.rs/bitcoin/latest/bitcoin/blockdata/fee_rate/struct.FeeRate.html#method.to_sat_per_kwu)(self) or [to_sat_per_vb_ceil](https://docs.rs/bitcoin/latest/bitcoin/blockdata/fee_rate/struct.FeeRate.html#method.to_sat_per_vb_ceil)(self).

Reply:
Yes, I am following #1141. Once it is in we can remove this.
It is your call if you want to add these tests before #1141.